pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
+
+- **Cloudlink Available, but heartbeat not**: The device is unable to communicate with the {{ $names.cloud.lower }} API. A device with cloudlink only is not able to apply any new changes made such as deploying new releases, applying service configuration values, or switching to local mode. However, it is accessible via SSH or the web terminal. Performing an action such as rebooting or restarting containers might work, but most likely will not. This is because the device loses its Heartbeat if it's not communicating with the API, which is usually when the Supervisor on the device is not running or crashing. Since we have cloudlink tunnel access, we can SSH into the device and investigate further.
 
-<sup>2</sup> `Online (Heartbeat Only)` indicates that device is unable to connect to cloudlink (e.g. a firewall is blocking VPN traffic), and won't be able to provide remote SSH connections. A device with a Heartbeat Only status has internet connectivity and can poll the cloud for new updates to apply. However, performing any actions which do not change the target state or attempting to access the device via SSH will error, as these actions only have effect with a VPN connection. Actions that do not change the target state include purging data, restarting, rebooting, or shutting down, and they all require the VPN because they are proxied from the dashboard API to the [Supervisor API](/reference/supervisor/supervisor-api/) on device.
+- **Heartbeat Available, but cloudlink not**: The device is unable to connect to cloudlink (e.g. a firewall is blocking VPN traffic), and won't be able to provide remote SSH connections. A device with Heartbeat Only status has internet connectivity and can poll the cloud for new updates to apply. However, performing any actions which do not change the target state or attempting to access the device via SSH will error, as these actions are proxied via the cloudlink tunnel connection. Actions that do not change the target state include purging data, restarting, rebooting, or shutting down.
 
-__Note:__ If the device is powered off or loses all network connectivity, it will remain in the `Online (Heartbeat Only)` state until the last target state fetch exceeds the device [API polling interval][poll-interval].
+__Note:__ If the device is powered off or loses all network connectivity, the `heartbeat` indictor show as connected (green) until the last target state fetch exceeds the device [API polling interval][poll-interval], at which time it will be marked as `Disconnected`.
 
 ## Debugging Device Status
 
 If you find your device to be displaying a status which is unclear even with the above notes, visit the [device debugging masterclass][debugging-masterclass] for more information.
 
```

Comment:
The padding here is a bit weird? It looks like 'Update Status' is within the 'Debugging Device Status' section. I'd suggest adding a bigger gap here.